cli/cmd/start.go

```diff
@@ -30,6 +31,21 @@ func NewStartCmd() *cobra.Command {
 				internalStartModule, args)
 			handleCmdErr(cmd, err)
 		},
+		ValidArgsFunction: func(
```

Comment:
Let;s avoid code duplication and make the code of this auto-completion function common.